docs/source/pages/supported_technologies/index.rst

```diff
@@ -107,6 +107,17 @@ Provenances
        * The provenance should be published on JFrog Artifactory
      - :doc:`page </pages/supported_technologies/jfrog>`
 
+.. _supported_automatic_deps_resolution:
+
+-------------------------------
+Automatic dependency resolution
+-------------------------------
+
+Currently, we support the following type of project for automatic dependency resolution.
+
+* Java Maven
+* Java Gradle
```

Comment:
We also support Python if the virtual environment path is provided and packages are built with Python 3.11.

Reply:
```
* Python (if a Python virtual environment is created and packages are installed using Python3.11, see :ref:` providing Python virtual environment <python-venv-deps>.)
```
How do you think about it ?

Reply:
Sounds good.

Reply:
Thanks. Added in https://github.com/oracle/macaron/pull/840/commits/87e7bb9f03e43260b91db2669a81008af5972fbc